docs/source/api/plugin/usage-reporting.md

```diff
@@ -321,9 +325,49 @@ If set, prints all reports as JSON when they are sent. (Note that this feature i
 </td>
 <td>
 
-Specify the function for creating a signature for a query. This option is not recommended, as Apollo's servers make assumptions about how the signature relates to the operation you executed.
+Specify this function to create a signature for a query. This option is not recommended, because Apollo's servers make assumptions about how the signature relates to the operation you executed.
 </td>
 </tr>
 
 </tbody>
 </table>
+
+#### Valid `sendHeaders` object signatures
```

Comment:
Move this below the `sendVariableValues` section so they are in the same order as the options in the table (and are in decreasing order of importance).

Reply:
I was trying so hard not to get these flipped and still, I got them flipped